deploy/esignet/delete.sh

```
@@ -8,11 +8,12 @@ fi
 
 function Deleting_esignet() {
   NS=esignet
+  ESIGNET_SERVICE_NAME=esignet
   while true; do
       read -p "Are you sure you want to delete all esignet helm charts?(Y/n) " yn
```

Shellcheck (0.11.0)

[info] read without -r will mangle backslashes.

(SC2162)